portfolio/src/main/java/com/google/sps/servlets/randomGet.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
```

Comment:
Can we be more specific with the file name? Perhaps RandomNumberServlet.java?

(also note: file names in Java are typically CamelCase, where even the first word is capitalized)

Reply:
Oh yeah I always confuse CamelCase in methods and files